plugins/inputs/http_listener_v2/http_listener_v2_test.go

```diff
@@ -49,7 +49,7 @@ func newTestHTTPListenerV2() *HTTPListenerV2 {
 	listener := &HTTPListenerV2{
 		Log:            testutil.Logger{},
 		ServiceAddress: "localhost:0",
-		Path:           "/write",
+		Paths:          []string{"/write"},
```

Comment:
Please keep those tests unmodified to check backward compatibility.

Reply:
Tests restored to original state